### DISCUSSION TITLE
Installation Problem of v4.8.2

### DISCUSSION THREAD

**Discussion by Willy1995** [iqm-finland/KQCircuits#99] [Q&A] [answered]
Hello,

I had installed KQCircuit salt package successfully at May, 9 (I forget the version at that time).

Today I update KQCircuit to v4.8.2. And it shows an error message. I show it below.

The KQCircuit I installed before is also vanished. The salt package file is empty now.

Do you know how to fix this problem?

![kqcircuit](https://github.com/iqm-finland/KQCircuits/assets/37732896/df3b8f91-a6ed-4d6a-aad3-8ca2e05833ca)

**Comment by qpavsmi** [comment, marked as answer]
Hi! Which version of KLayout are you currently using? If it's anything earlier than 0.29.0 I highly recommend to install the newest KLayout version. There was a recent change in KLayout Salt packaging system where it currently uses GIT instead of SVN to fetch plugin packages.

If this doesn't fix it, let me know which operating system your machine uses.

I also want to remind you that SALT installation of KQCircuits is designed for demonstration and introduction purposes. If you plan to do actual design work using KQCircuits, we highly recommend to at some point follow the develop setup guide [here](https://iqm-finland.github.io/KQCircuits/developer/setup.html) and [here](https://iqm-finland.github.io/KQCircuits/developer/standalone.html) for more versatility in using KQCircuits.

**Reply by Willy1995** [reply]
Hello,

I use KLayout version 0.29.2 and Window 10. Although I still meet the same problem on the salt package, I succeessfully install KQCircuit by using the developer setup.

Thank you very much!
